torch/csrc/jit/script/module.h

```diff
@@ -224,34 +226,12 @@ struct TORCH_API Module {
         });
   }
 
-  c10::optional<Slot> find_parameter(const std::string& name) const {
-    return find_slot(name, EntityType::PARAMETER);
-  }
-  c10::optional<Slot> find_attribute(const std::string& name) {
-    return find_slot(name, EntityType::ATTRIBUTE);
-  }
-  c10::optional<Slot> find_buffer(const std::string& name) {
-    auto iv = find_attribute(name);
-    if (iv && iv->type()->isSubtypeOf(TensorType::get())) {
-      return iv;
-    }
-    return c10::nullopt;
-  }
-  c10::optional<Module> find_module(const std::string& name) const {
-    if (auto slot = find_slot(name, EntityType::MODULE)) {
-      return Module(slot->value().toObject());
-    }
-    return c10::nullopt;
-  }
-  c10::optional<Method> find_method(const std::string& basename) const {
-    for (Function* fn : type()->methods()) {
-      if (fn->name() == basename) {
-        return Method(module_object(), fn);
-      }
+  c10::optional<size_t> find_parameter(const std::string& name) const;
```

Comment:
I don't like this change; it complicates to a common operation (looking up stuff on a module), and it seems like if `get_x` returns `T`, `find_x` should return `optional<T>` for least surprise.